framework/src/problems/FEProblemBase.C

```diff
@@ -221,7 +221,55 @@ FEProblemBase::FEProblemBase(const InputParameters & parameters)
+        registerTimedSection("reinitBecauseOfGhostingOrNewGeomObjects", 3)),
+    _exec_multi_app_transfers_timer(registerTimedSection("execMultiAppTransfers", 1)),
+    _init_timer(registerTimedSection("init", 2)),
+    _eq_init_timer(registerTimedSection("EquationSystems::Init", 2)),
+    _solve_timer(registerTimedSection("solve", 1)),
+    _check_exception_and_stop_solve_timer(registerTimedSection("checkExceptionAndStopSolve", 5)),
+    _advance_state_timer(registerTimedSection("advanceState", 5)),
+    _restore_solutions_timer(registerTimedSection("restoreSolutions", 5)),
+    _save_old_solutions_timer(registerTimedSection("saveOldSolutions", 5)),
+    _restore_old_solutions_timer(registerTimedSection("restoreOldSolutions", 5)),
+    _output_step_timer(registerTimedSection("outputStep", 1)),
+    _on_timestep_begin_timer(registerTimedSection("onTimestepBegin", 2)),
+    _compute_residual_l2_norm_timer(registerTimedSection("computeResidualL2Norm", 2)),
+    _compute_residual_sys_timer(registerTimedSection("computeResidualSys", 5)),
+    _compute_residual_internal_timer(registerTimedSection("computeResidualInternal", 1)),
+    _compute_residual_type_timer(registerTimedSection("computeResidualType", 5)),
+    _compute_transient_implicit_residual_timer(
+        registerTimedSection("computeTransientImplicitResidual", 2)),
+    _compute_residual_tags_timer(registerTimedSection("computeResidualTags", 5)),
+    _compute_jacobian_internal_timer(registerTimedSection("computeJacobianInternal", 1)),
+    _compute_jacobian_tags_timer(registerTimedSection("computeJacobianTags", 5)),
+    _compute_jacobian_blocks_timer(registerTimedSection("computeTransientImplicitJacobian", 2)),
+    _compute_bounds_timer(registerTimedSection("computeBounds", 1)),
+    _compute_post_check_timer(registerTimedSection("computePostCheck", 2)),
+    _compute_damping_timer(registerTimedSection("computeDamping", 1)),
+    _possibly_rebuild_geom_search_patches_timer(
+        registerTimedSection("possiblyRebuildGeomSearchPatches", 5)),
+    _initial_adapt_mesh_timer(registerTimedSection("initialAdaptMesh", 2)),
+    _adapt_mesh_timer(registerTimedSection("adaptMesh", 3)),
+    _update_mesh_xfem_timer(registerTimedSection("updateMeshXFEM", 5)),
+    _mesh_changed_timer(registerTimedSection("meshChanged", 3)),
+    _mesh_changed_helper_timer(registerTimedSection("meshChangedHelper", 5)),
+    _check_problem_integrity_timer(registerTimedSection("notifyWhenMeshChanges", 5)),
+    _serialize_solution_timer(registerTimedSection("serializeSolution", 3)),
+    _check_nonlinear_convergence_timer(registerTimedSection("checkNonlinearConvergence", 5)),
+    _check_linear_convergence_timer(registerTimedSection("checkLinearConvergence", 5)),
+    _update_geometric_search_timer(registerTimedSection("updateGeometricSearch", 3)),
+    _exec_multi_apps_timer(registerTimedSection("execMultiApps", 3)),
+    _backup_multi_apps_timer(registerTimedSection("backupMultiApps", 5))
```

Comment:
Hmmm - I think that would look just as messy probably - because you still have to give them names, etc.  So you would just have one really long line instead of a bunch of small ones...

This really only happens in FEProblem... because it's freaking huge!